apiary.apib

```diff
@@ -4108,6 +4118,11 @@ A partial update of a data plane, one of the attributes are required.
 + host: demo.snowflakecomputing.com (required)
 + backend: snowflake (required)
 + warehouse: keboola
++ saml2Configuration:
```

Comment:
nechces pripsat kus vety o tom co to je? treba aspon ze to je jen pro SNFLK

Reply:
No to do té sračky ani nejde rozumně zapsat nebo jakože možu to tam prcnůt ono to v tom UI stejně vidět není 
![image](https://github.com/user-attachments/assets/93497ad6-0f52-479e-a238-3ce35f773dce)

Reply:
ale jo, sice az pri rozkliknuti, ale jo
![image](https://github.com/user-attachments/assets/45d23320-2f6b-4bc6-bb68-16838013eeef)

Reply:
ok přidal sem